### PR TITLE
Deprecated v9 test vectors

### PR DESCRIPTION
**Pull request by HunterEwok**
Description for V9 moved to TestVectors-v9.md
Removed V9 related info from the main page

### CODE REVIEW

**Review by sanchi21 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ These test vectors aim to conform to the latest industry standards, among these
 
 * [v6](https://github.com/Axinom/dash-test-vectors/tree/conservative) - a conservative set of test vectors that utilized less modern features for wider compatibility (for the time).
 * [v7 & v8](TestVectors-v7-v8.md) - a set of tests vectors that (for the time) intentionally utilized more advanced features - designed to push the boundaries of implementations and to encourage uptake of modern features.
+* [v9](TestVectors-v9.md) - a set of tests vectors that (for the time) intentionally utilized more advanced features - designed to push the boundaries of implementations and to encourage uptake of modern features.
```
Comment (Heronyme) [inline README.md]: I would replace this just with: the last set of test vectors packaged by Axinom in-house packager "Makemedia".

**Comment by Heronyme** [comment]
Please also adjust PR title a bit, e.g. to: "Deprecated v9 test vectors".

**Review by Heronyme — CHANGES_REQUESTED** [review]

**Review by vikneswaranrr — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (HunterEwok) [reply README.md]: Fixed

**Review by Heronyme — APPROVED** [review]
Looks good

**Review by sakvith — APPROVED** [review]

**Pull request merged**